### ISSUE TITLE
The font in `acl_lualatex.tex' is incompatible with aclpubcheck

### ISSUE THREAD

**Issue by max-ionov** [acl-org/acl-style-files#62]
I am not sure if this should be reported here or in the [aclpubcheck repository](https://github.com/acl-org/aclpubcheck), but it seems that `aclpubcheck` font requirements are more up-to-date than in the example here, so probably the example file should be corrected.

The issue: `acl_lualatex.tex` [uses](https://github.com/acl-org/acl-style-files/blob/12f3d63b1d3a9ae4bd6c59638ee5f89aa9169592/latex/acl_lualatex.tex#L22) TeX Gyre Termes font from the package [tex-gyre](https://ctan.org/pkg/tex-gyre). `aclpubcheck`, the checker tool that makes sure that PDFs are compatible with the guidelines throws an error on the example file:
```
Wrong font. The main font used is UXCRMS+TeXGyreTermes-Regular when it should a font in {'ICZIZQ+Inconsolatazi4-Regular', 'ICWANT+STIXGeneral-Regular', 'NimbusRomNo9L-Regu', 'TeXGyreTermesX-Regular', 'TimesNewRomanPSMT'}.
```

A solution to this is to change the [line linked above](https://github.com/acl-org/acl-style-files/blob/12f3d63b1d3a9ae4bd6c59638ee5f89aa9169592/latex/acl_lualatex.tex#L22) to use the TeX Gyre Termes X font from the package [newtx](https://ctan.org/pkg/newtx):

```tex
\babelfont{rm}{TeXGyreTermesX}
```
